src/components/CompareResults/RevisionRow.tsx

```diff
@@ -65,6 +102,18 @@ function RevisionRow(props: RevisionRowProps) {
     setExpanded(!expanded);
   };
 
+  // Note that the return type is different depending on the view we're in
+  const loaderData = useLoaderData() as
+    | WithBaseLoaderReturnValue
+    | OverTimeLoaderReturnValue;
+  const subtestsCompareLink =
+    loaderData.view === compareView
+      ? getSubtestsCompareWithBaseLink(result)
+      : getSubtestsCompareOverTimeLink(
+          result,
+          (loaderData as OverTimeLoaderReturnValue).intervalValue,
```

Comment:
@julienw I'm not sure about this change - using `loaderData.internalValue` is causing typescript error _Property does not exist on type_

Reply:
By changing the condition and inverting the ternary, this works:
```js
  const subtestsCompareLink =
    loaderData.view === compareOverTimeView
      ? getSubtestsCompareOverTimeLink(result, loaderData.intervalValue)
      : getSubtestsCompareWithBaseLink(result);
```

There might be something happening with the WithBaseLoaderReturnValue type, but let's not waste time on that.

Reply:
I still get the same error even with these changes.l